test/boost/expr_test.cc

```diff
@@ -1322,17 +1322,20 @@ BOOST_AUTO_TEST_CASE(prepare_token) {
                                   .build();
     auto [db, db_data] = make_data_dictionary_database(table_schema);
 
-    expression tok =
-        token({::make_shared<column_identifier_raw>("p1", true), ::make_shared<column_identifier_raw>("p2", true),
-               ::make_shared<column_identifier_raw>("p3", true)});
-
-    expression prepared = prepare_expression(tok, db, "test_ks", table_schema.get(), nullptr);
+    expression tok = token({unresolved_identifier{.ident = ::make_shared<column_identifier_raw>("p1", true)},
+                            unresolved_identifier{.ident = ::make_shared<column_identifier_raw>("p2", true)},
+                            unresolved_identifier{.ident = ::make_shared<column_identifier_raw>("p3", true)}});
```

Comment:
Unrelated: we can drop shared_ptr from column_identifier_raw, it's just obfuscating the code.